pkg/services/provisioning/dashboards/config_reader.go

```diff
@@ -88,6 +89,10 @@ func (cr *configReader) readConfig() ([]*config, error) {
 			dashboard.OrgID = 1
 		}
 
+		if err := utils.CheckOrgExists(dashboard.OrgID); err != nil {
+			return nil, fmt.Errorf("failed to provision dashboards with %s reader: %w", dashboard.Name, err)
```

Comment:
I think it's better to quote the dashboard name, in case it contains whitespace.
```suggestion
			return nil, fmt.Errorf("failed to provision dashboards with %q reader: %w", dashboard.Name, err)
```